src/bin/nydusd/main.rs

```diff
@@ -717,6 +717,11 @@ fn main() -> Result<()> {
     let apisock = args.value_of("apisock");
 
     setup_logging(logging_file, level)?;
+
+    log_panics::Config::new()
```

Comment:
Maybe move it into `setup_logging`, so that enables the panic hook for `nydus-image`, `nydusd`, `nydus-cached`.

Reply:
good point